### PR TITLE
v11 color token updates for select component

### PR DESCRIPTION
**Pull request by kingtraceyj**
This PR updates the color tokens on the Style tab for the select component.

### CODE REVIEW

File: src/pages/components/select/style.mdx [hunk 1]
```diff
@@ -12,15 +12,15 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
-| `.bx--select-input`   | background | `$field-01` |
-| `.bx--select--light`  | background | `$field-02` |
-| `.bx--select--inline` | background | transparent |
-| `.bx--label`          | text color | `$text-01`  |
-| `.bx--select-input`   | text color | `$text-01`  |
-| `.bx--select--inline` | text color | `$icon-01`  |
-| `.bx--select__arrow`  | fill       | `$icon-01`  |
+| Class                 | Property   | Color token                 |
+| --------------------- | ---------- | --------------------------- |
+| `.bx--select-input`   | background | `$background`               |
```
Comment (aagonzales) [inline src/pages/components/select/style.mdx:17]: The would be `$field`

File: src/pages/components/select/style.mdx [hunk 2]
```diff
@@ -12,15 +12,15 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
+| `.bx--select--light`  | background | `$background: inline theme` |
```
Comment (aagonzales) [inline src/pages/components/select/style.mdx:18]: And you can delete this row. We're not showing the `light` variants anymore. You'd only us "inline theme" if the components require that version for the default style, like within Modal.

File: src/pages/components/select/style.mdx [hunk 3]
```diff
@@ -32,13 +32,13 @@ input color is `$field-02` and is used on `$ui-01` page backgrounds.
...
-| `.bx--select-input[data-invalid]` | border     | `$support-01`  |
-| `.bx--form-requirement`           | text color | `$support-01`  |
-| `.bx--select-input:disabled`      | background | `$disabled-01` |
-| `.bx--select-input:disabled`      | text color | `$disabled-02` |
+| Class                             | Property   | Color token            |
+| --------------------------------- | ---------- | ---------------------- |
+| `.bx--select-input:focus`         | border     | `$focus`               |
+| `.bx--select-input[data-invalid]` | border     | `$support-error`       |
+| `.bx--form-requirement`           | text color | `$support-error`       |
+| `.bx--select-input:disabled`      | background | `$background-disabled` |
```
Comment (aagonzales) [inline src/pages/components/select/style.mdx:40]: This would be `$field-disabled`

**Review by aagonzales — CHANGES_REQUESTED** [review]

**Review by aagonzales — APPROVED** [review]

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Pull request merged**